x/delayedack/genesis.go

```diff
@@ -9,9 +9,8 @@ import (
 // InitGenesis initializes the module's state from a provided genesis state.
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
 	k.SetParams(ctx, genState.Params)
-	// Validate all other genesis fields are empty
-	if len(genState.RollappPackets) != 0 {
-		panic("Only params can be initialized at genesis")
+	for _, packet := range genState.RollappPackets {
+		k.SetRollappPacket(ctx, packet)
```

Comment:
i think we need to re-add the tests for init and export genesis for the delayedack module

Reply:
I see that we don't have genesis_test for delayedack module before.

Reply:
cc: @srene